src/utils/dateTime.ts

```diff
@@ -36,6 +36,10 @@ export const isInFuture = (date: Posix): boolean => {
   return date > Date.now()
 }
 
+export const isOlderThan14Days = (date: Posix): boolean => {
```

Comment:
I might keep this one colocated with where it is used, i dont think we know if this will need to support multiple callers at this point.